AcademyApp/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  ViewController.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 13/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController
+{
+    // MARK: - View Controller's life cycle methods
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addObservers()
+        setupUI()
+        emailTextField.delegate = self
+        passwordTextField.delegate = self
+        
```

Comment:
delegates setup can be moved to setup methods as well

Reply:
Moved delegation setup to setup methods.